app/packages/looker/src/elements/common/tags.ts

```diff
@@ -321,14 +321,12 @@ export class TagsElement<State extends BaseState> extends BaseElement<State> {
         if (field && LABEL_RENDERERS[field.embeddedDocType]) {
           if (path.startsWith("frames.")) continue;
           const classifications = LABEL_LISTS.includes(field.embeddedDocType);
-
-          if (!value.classifications?.length) continue;
```

Comment:
I know this caused other regressions. bringing it back the None bubble issue - I'll fix it another way